luigi.cfg

```diff
@@ -0,0 +1,16 @@
+[core]
+# for individual machine
+default-scheduler-url = http://localhost:8082/
+# for High Performance Cluster (lsf script, bsub command)
+# default-scheduler-url = http://cmu166.research.partners.org:8082/
+
```

Comment:
We should dedicate a virtual machine (VM) from Partners where we can launch Luigi server. The `cmu166` node behaved unstably in our recent experiment. A Luigi task failed to connect to the scheduler. Other modifications may include setting `max_retry` and `timeout` parameters to a value higher than their defaults.